server/build-osx.sh

```diff
@@ -243,6 +244,14 @@ _build_server_osx() {
     ssh $PG_SSH_OSX "cp -pR /opt/local/Current_v15/lib/libicudata*dylib $PG_STAGING/lib/" || _die "Failed to copy the latest libicudata"
     ssh $PG_SSH_OSX "cp -pR /opt/local/Current_v15/lib/libicuuc*dylib $PG_STAGING/lib/" || _die "Failed to copy the latest libicuuc"
     ssh $PG_SSH_OSX "cp -pR /opt/local/Current_v15/lib/libiconv*.dylib $PG_STAGING/lib/" || _die "Failed to copy the latest libiconv"
+    ssh $PG_SSH_OSX "cp -pR /opt/local/Current_v15/lib/libkrb5*.dylib $PG_STAGING/lib/" || _die "Failed to copy the latest libkrb5"
+    ssh $PG_SSH_OSX "cp -pR /opt/local/Current_v15/lib/libgss*.dylib $PG_STAGING/lib/" || _die "Failed to copy the latest libgssapi"
+    ssh $PG_SSH_OSX "cp -pR /opt/local/Current_v15/lib/libk5*.dylib $PG_STAGING/lib/" || _die "Failed to copy the latest libk5"
+    ssh $PG_SSH_OSX "cp -pR /opt/local/Current_v15/lib/libverto*.dylib $PG_STAGING/lib/" || _die "Failed to copy the latest libverto"
+    ssh $PG_SSH_OSX "cp -pR /opt/local/Current_v15/lib/libcom*.dylib $PG_STAGING/lib/" || _die "Failed to copy the latest libcom"
+    ssh $PG_SSH_OSX "cp -pR /opt/local/Current_v15/lib/libkdb5*.dylib $PG_STAGING/lib/" || _die "Failed to copy the latest libkdb5"
+    ssh $PG_SSH_OSX "cp -pR /opt/local/Current_v15/lib/libkadm5*.dylib $PG_STAGING/lib/" || _die "Failed to copy the latest libkadm5"
+    ssh $PG_SSH_OSX "cp -pR /opt/local/Current_v15/lib/libkrad*.dylib $PG_STAGING/lib/" || _die "Failed to copy the latest libkrad"
```

Comment:
hope all these dylibs are required and we are not necessarily bundling the ones which are not needed?
please also update 3rd party license to include krb5